indico/MaKaC/webinterface/pages/abstracts.py

```diff
@@ -430,6 +431,7 @@ def getVars(self):
         vars["abstractTitle"] = quoteattr(str(vars.get("title", "")))
         vars["prAuthors"] = fossilize(vars.get("prAuthors", []))
         vars["coAuthors"] = fossilize(vars.get("coAuthors", []))
+        vars["authors"] = fossilize(getAuthorsFromAuthorIndex(self._conf, 10))
```

Comment:
Calling this `"authors"` is a bit dangerous. `prAuthors and `coAuthors` above serve a completely different purpose: they are the initial values for the fields. Why not `suggested_authors`?